staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -28,20 +28,36 @@ const (
 	StatefulSetRevisionLabel       = ControllerRevisionHashLabelKey
 	DeprecatedRollbackTo           = "deprecated.deployment.rollback.to"
 	DeprecatedTemplateGeneration   = "deprecated.daemonset.template.generation"
-	StatefulSetPodNameLabel        = "statefulset.kubernetes.io/pod-name"
+	// StatefulSetPodNameLabel corresponds to the unique name of each Pod.
+	// More info: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-name-label
+	StatefulSetPodNameLabel = "statefulset.kubernetes.io/pod-name"
```

Comment:
Can pod name be longer than the allowed label length?

Reply:
Why do we need this label when you can already get pod name in downward API?

Reply:
>Why do we need this label when you can already get pod name in downward API?

I believe `statefulset.kubernetes.io/pod-name` is applied in order to support creating a unique service per pod. But please note that this changeset doesn't add the pod-name label; I only added a comment for the existing `StatefulSetPodNameLabel` const.

> Can pod name be longer than the allowed label length?

Yes, and this is a common (existing) problem with statefulsets. See https://github.com/kubernetes/kubernetes/issues/64023.

Reply:
Ok, let's remove all pod name references in here and the docs then.